docs/internals/protocol/typedesc.rst

```diff
@@ -23,6 +23,11 @@ to binary data that EdegDB can decode and work with.
 A *decoder* is used to decode data from EdgeDB native format to
 data types native to the driver.
 
+There is one special type with *type id* of zero:
+``00000000-0000-0000-0000-000000000000``. When you describe this type, result
+contains zero *blocks*. It's used when statement returns no meaningful results,
+for example for ``CREATE DATABASE example`` statement.
```

Comment:
```suggestion
e.g. the ``CREATE DATABASE example`` statement.
```

Reply:
Well, actually is there any difference?

Reply:
I think stylistically we should avoid using `e.g.`, `i.e.`, `etc`.  "for example, the `CREATE DATABASE foo` statement" would read better